crates/blockchain/error.rs

```diff
@@ -34,6 +34,8 @@ pub enum InvalidBlockError {
     GasUsedMismatch,
     #[error("Blob gas used doesn't match value in header")]
     BlobGasUsedMismatch,
+    #[error("Block marked as invalid")]
+    AlreadyInvalid,
```

Comment:
Do we need the concepto of "already"? can't it just be invalid?

Reply:
Removed entirely.